components/html-block/test/html-block.visual-diff.js

```diff
@@ -42,7 +42,9 @@ describe('d2l-html-block', () => {
 		it(info.name, async function() {
 			const rect = await visualDiff.getRect(page, info.selector);
 			if (info.action) await info.action(info.selector);
-			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+			setTimeout(async() => {
```

Comment:
I'm not sure the test will actually wait for the screenshot.  You might need something like:
```javascript
await new Promise(resolve => setTimeout(resolve, 0));
await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
```

Reply:
Ah, of course.